### PR TITLE
Document temporality presets in Python

### LINKED ISSUE #2549: Document temporality presets in Python
The missing temporality preset `LowMemory` [was added here](https://github.com/open-telemetry/opentelemetry-python/pull/3223). Add documentation for all the presets.

### PR DESCRIPTION
**Pull request by ocelotl**
- Fixes #2549

---

Preview: https://deploy-preview-2551--opentelemetry.netlify.app/docs/instrumentation/python/exporters/#temporality-presets

### CODE REVIEW

File: content/en/docs/instrumentation/python/exporters.md [hunk 1]
```diff
@@ -43,6 +43,52 @@ different interval, change the
...
 kind.
 
+#### Temporality Presets
+
+There are temporality presets for each instrumentation kind. These presets can
+be set with the environment variable
+`OTEL_EXPORTER_METRICS_TEMPORALITY_PREFERENCE`, for example:
+
+```console
+$ export OTEL_EXPORTER_METRICS_TEMPORALITY_PREFERENCE="DELTA"
```
Comment (chalin) [inline content/en/docs/instrumentation/python/exporters.md:53]:
```suggestion
```sh
export OTEL_EXPORTER_METRICS_TEMPORALITY_PREFERENCE="DELTA"
```

(Note: you don't actually need the quotes in the above statement, but it's ok if you want to keep them.)

File: content/en/docs/instrumentation/python/exporters.md [hunk 2]
```diff
@@ -43,6 +43,52 @@ different interval, change the
...
+```
+
+The default value for `OTEL_EXPORTER_METRICS_TEMPORALITY_PREFERENCE` is
+`"CUMULATIVE"`.
```
Comment (chalin) [inline content/en/docs/instrumentation/python/exporters.md:57]:
```suggestion
`CUMULATIVE`.
```

File: content/en/docs/instrumentation/python/exporters.md [hunk 3]
```diff
@@ -43,6 +43,52 @@ different interval, change the
...
+  - `Counter`: `DELTA`
+  - `UpDownCounter`: `CUMULATIVE`
+  - `Histogram`: `DELTA`
+  - `ObservableCounter`: `CUMULATIVE`
+  - `ObservableUpDownCounter`: `CUMULATIVE`
+  - `ObservableGauge`: `CUMULATIVE`
+
+Setting `OTEL_EXPORTER_METRICS_TEMPORALITY_PREFERENCE` to any other value than
+`CUMULATIVE`, `DELTA` or `LOWMEMORY` will log a warning and set this environment
+variable to `CUMULATIVE`.
```
Comment (chalin) [inline content/en/docs/instrumentation/python/exporters.md:90]:
```suggestion
If you set `OTEL_EXPORTER_METRICS_TEMPORALITY_PREFERENCE` to a value other than
`CUMULATIVE`, `DELTA`, or `LOWMEMORY`, then a warning will be logged and the variable
value will fall back to the default of `CUMULATIVE`.
```

You might want to say how/what is logging the warning?

**Review by chalin — COMMENTED** [review]
Thanks. See inline comments.

**Review by cartermp — APPROVED** [review]
Thanks!

**Pull request merged**